cfgov/jinja2/v1/_layouts/content-base-main-first.html

```diff
@@ -12,7 +12,10 @@
 {% block body_content scoped %}
 <div class="wrapper content_wrapper">
     <div class="content_main {% block content_main_modifiers -%}{%- endblock %}">
-        <section>
+        <section lang="{{ page.language }}"
```

Comment:
You wouldn't have to change the code in three places if we wanted to add a language which was `rtl`.  We should also consider just returning the direction so we wouldn't need the if statements. 

Like:
`page.language.name` and `page.language.direction`